### ISSUE TITLE
probably should warn about async tests

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/eslint-plugin-cypress#58]
```js
it('should warn', async () => {
  ...
})
```

**Comment by E-Kuerschner** [comment]
Yes, yes, yes! I installed this lib hoping to find support for this. I would be willing to give this a shot if no other contributors show interest.
